elixir/apps/domain/priv/repo/migrations/20231107154358_backfill_routing_column.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Domain.Repo.Migrations.BackfillRoutingColumn do
+  use Ecto.Migration
+  import Ecto.Query
+
+  @disable_ddl_transaction true
+  @disable_migration_lock true
```

Comment:
For the time being we should avoid disabling DDLs and migration locks, this gets tricky in production because of how our migrations are run - every node in a cluster attempts to run them when it boots. This means that, without locks, two nodes can attempt to run them concurrently and deadlock each other or at the very least send one of them to crash-reboot.

So if we will have a ton of data and really need this in future we also will have to build a central migration process.

Reply:
Ok, I was just trying to follow some of the best practices that I read, but I can get rid of this.